ui/components/configuratorComponents/patternConfigurator.js

```diff
@@ -45,6 +45,12 @@ const useStyles = makeStyles((theme) => ({
       height : '100%',
     }
   },
+  patterntype : {
```

Comment:
```suggestion
  patternType : {
```

Reply:
This is not a required change, but my OCD kinda made me put it here 😶 

CC @Chadha93

Reply:
@adithyaakrishna Absolutely, committing this change :smile: